### PR TITLE
Feat/add linter

### PR DESCRIPTION
**Pull request by simon-the-shark**
so this sets up the linter, typecheck, prettiercheck ci workflow. Simple tests should work too, but if you need db in tests, then you'll probably need to set up it too (unless adonis has some built in mechanism for this shit, I have no clue)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,10 +2,11 @@ TZ=UTC
...
+APP_KEY=nice_keynice_keynice_keynice_keynice_keynice_key
 NODE_ENV=development
 DB_HOST=127.0.0.1
 DB_PORT=5432
 DB_USER=root
 DB_PASSWORD=root
-DB_DATABASE=app
\ No newline at end of file
+DB_DATABASE=app
+LOG_LEVEL=debug
```
Comment (czaja307) [inline .env.example:12]: why there are 2 LOG_LEVELs?

Comment (simon-the-shark) [reply .env.example:12]: good question

Comment (simon-the-shark) [reply .env.example:12]: lets  do a quick check

Comment (czaja307) [reply .env.example]: it seems to work ig

**Review by czaja307 — APPROVED** [review]

**Pull request merged**